apps/api/src/app/vehikl/team-member/TeamMember.ts

```diff
@@ -11,31 +12,50 @@ registerEnumType(Office, {
   description: 'Office Location'
 });
 
+const id_description = 'Team Member unique id within their office';
```

Comment:
whats the benefits of having them as `consts` up here?

Reply:
I don't have to repeat the same string for the description of the field for swagger AND graphql